packages/http/src/stamp.ts

```diff
@@ -62,6 +69,23 @@ async function signMessage(
   return uint8ArrayToHexString(signatureDer);
 }
 
+function convertTurnkeyPrivateKeyToJwk(input: {
+  uncompressedPrivateKeyHex: string;
+  compressedPublicKeyHex: string;
+}): JsonWebKey {
```

Comment:
We should contemplate changing the API key format if jwk is easier to deal with. We could change the CLI so that it outputs new API keys in JWK, and have a feature to migrate from one format to another. We're still early enough that it'd be doable!

The checked-in `tink` dependency is definitely reasonable though. The only "bad" thing about it is it's a tiny bit too broad. It supports P256, P384, and P521 but we only need P256. Even with that, the size isn't bad and the code is readable so I'm not opposed to taking it on.